Library/Formula/shc.rb

```diff
@@ -0,0 +1,23 @@
+class Shc < Formula
+  desc "Shell Script Compiler"
+  homepage "https://neurobin.github.io/shc"
+  url "https://github.com/neurobin/shc/archive/3.9.3a.tar.gz"
```

Comment:
It's a trivial change. If you're eager to get this formula in, you could just change this to use the stable 3.9.2 and stick a `chmod` call inside the `def install`. Otherwise, do you want to close this issue out and re-submit when the real 3.9.3 is released?